cmd/kubelet/app/options/osflags_windows.go

```diff
@@ -24,4 +24,10 @@ import (
 
 func (f *KubeletFlags) addOSFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&f.WindowsService, "windows-service", f.WindowsService, "Enable Windows Service Control Manager API integration")
+	// The default priority class associated with any process in Windows is NORMAL_PRIORITY_CLASS. Keeping it as is
+	// to maintain backwards compatibility.
+	// Source: https://docs.microsoft.com/en-us/windows/win32/procthread/scheduling-priorities
+	fs.StringVar(&f.WindowsPriorityClass, "windows-priorityclass", "NORMAL_PRIORITY_CLASS",
```

Comment:
this is fine for windows, i wonder if it would help to correlate the priority class of actual end-user workloads themselves so users can make an informed choice?  for example, 'the default priority class on windows containers is X and could be elevated or altered via Y'?

Reply:
As of now, I don't think we have a mechanism to surface the priority class of a process(container) on Windows node to end user. 

`'the default priority class on windows containers is X and could be elevated or altered via Y'?`

The only field I can think of in the pod spec is pod's priority class to express it, there are multiple ways to  modify the priority on the node but I am not sure if we need it at the moment unless you've a use-case in mind.